components/post_view/post_attachment_opengraph/post_attachment_opengraph.tsx

```diff
@@ -41,230 +47,212 @@ export type Props = {
+                description={openGraphData?.description}
+            />
+            <PostAttachmenOpenGraphImage
+                src={imageUrl}
+                title={openGraphData?.title}
+                isInPermalink={rest.isInPermalink}
+                large={isLargeImage}
+                collapsed={!showImagePreview}
+                toggleImagePreviewHandler={toggleImagePreview}
+            />
+        </a>
+    );
+};
 
-        return (
-            <div className='attachment attachment--opengraph'>
-                <div className={classNames('attachment__content', permalinkClass)}>
-                    <div className={classNames('clearfix', 'attachment__container', 'attachment__container--opengraph', permalinkClass)}>
-                        <div className={'attachment__body__wrap attachment__body__wrap--opengraph'}>
-                            {!this.props.isInPermalink && <span className='sitename'>
-                                {this.truncateText(data.site_name)}
-                            </span>}
-                            {removePreviewButton}
-                            <h1 className={'attachment__title attachment__title--opengraph' + (data.title ? '' : ' is-url')}>
-                                <a
-                                    className='attachment__title-link attachment__title-link--opengraph'
-                                    href={makeUrlSafe(data.url || this.props.link)}
-                                    target='_blank'
-                                    rel='noopener noreferrer'
-                                    title={data.title || data.url || this.props.link}
-                                >
-                                    {this.truncateText(data.title || data.url || this.props.link)}
-                                </a>
-                            </h1>
-                            {body}
-                        </div>
-                        {!this.props.isInPermalink && imageUrl && !hasLargeImage && this.renderSmallImage(imageUrl, imageMetadata)}
-                    </div>
-                </div>
-            </div>
-        );
-    }
+type BodyProps = {
+    title: string;
+    isInPermalink?: boolean;
+    siteName?: string;
+    description?: string;
 }
 
-export function getBestImageUrl(openGraphData?: OpenGraphMetadata, imagesMetadata?: Record<string, PostImage>) {
-    if (!openGraphData?.images?.length) {
-        return undefined;
-    }
-
-    // Get the dimensions from the post metadata if they weren't provided by the website as part of the OpenGraph data
-    const images = openGraphData.images.map((image: OpenGraphMetadataImage) => {
-        const imageUrl = image.secure_url || image.url;
+const PostAttachmentOpenGraphBody = memo(({title, isInPermalink, siteName = '', description = ''}: BodyProps) => {
+    return (
+        <div className={classNames('PostAttachmenOpenGraph__body', {inPermalink: isInPermalink})}>
```

Comment:
This is total nitpicking, but could you change it so that we're consistent with the names of `inPermalink` and `isInPermalink`? Having them named differently seems like it'll lead to mistakes down the line